backend/bottari/src/main/java/com/bottari/teambottari/dto/ReadTeamMemberNameResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.bottari.teambottari.dto;
+
+import com.bottari.member.domain.Member;
+import com.bottari.teambottari.domain.TeamMember;
+
+public record ReadTeamMemberNameResponse(
+        Long id,
+        String name
+) {
+
+    public static ReadTeamMemberNameResponse of(final TeamMember teamMember) {
+        final Member member = teamMember.getMember();
+        return new ReadTeamMemberNameResponse(member.getId(), member.getName());
```

Comment:
```suggestion
    public static ReadTeamMemberNameResponse of(final TeamMember teamMember) {
        final Member member = teamMember.getMember();

        return new ReadTeamMemberNameResponse(member.getId(), member.getName());
```

이거 return 위에 띄워주는 설정 없나